src/python/bot/fuzzers/afl/engine.py

```diff
@@ -12,13 +12,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """AFL engine interface."""
+import os
+import stat
 
+from bot.fuzzers import engine_common
 from bot.fuzzers.afl import launcher
+from bot.fuzzers.afl import stats
 from lib.clusterfuzz.fuzz import engine
+from system import environment
+
+
+def _get_command_from_fuzz_result(fuzz_result, runner):
+  """Get the command string without showing unnecessary minijail commands."""
+  command = fuzz_result.command
+  if environment.get_value('USE_MINIJAIL'):
```

Comment:
nit: MINIJAIL is now completely disabled, so can skip this.

Reply:
Really? What replaced it for misbehaved OSS-Fuzz targets like rar?

Reply:
No replacement. @inferno-chromium disabled it and was monitoring.